src/resources/pvc.rs

```diff
@@ -9,7 +9,7 @@ pub enum StacksDevnetPvc {
 impl fmt::Display for StacksDevnetPvc {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
-            StacksDevnetPvc::StacksBlockchainApiPg => write!(f, "stacks-blockchain-api-pg"),
+            StacksDevnetPvc::StacksBlockchainApiPg => write!(f, "stacks-blockchain-api"),
```

Comment:
I see there are plenty of places elsewhere in this PR where `StacksDevnetPvc::StacksBlockchainApiPg` was removed, except here. Just double checking if this is intended? If so, I'm assuming `StacksDevnetPvc::StacksBlockchainApiPg` is being used for deleting the PVC on cleanup?

Reply:
Yes, this is on purpose!

When we switched to using the sts, I thought we'd be able to delete all of the PVC code, but as you pointed out, deletion of PVCs is not automatic, so we need to keep around some of this code.

Now, because this PVC is deployed by the sts, it doesn't have the same (or a predictable) name. Instead, we have to find it by label, and it inherits its label from the stateful set. That's why we're naming it as such here.